zanata-war/src/main/java/org/zanata/rest/editor/MediaTypes.java

```diff
@@ -32,6 +32,11 @@ public String toString() {
     public static final String APPLICATION_ZANATA_LOCALES_JSON =
             APPLICATION_ZANATA_LOCALES + JSON;
 
+    public static final String APPLICATION_ZANATA_SUGGESTIONS =
+            APPLICATION_VND_ZANATA + ".suggestions";
+    public static final String APPLICATION_ZANATA_SUGGESTIONS_JSON =
+            APPLICATION_ZANATA_SUGGESTIONS + JSON;
+
```

Comment:
Custom media types are kind of an open question in RESTful architecture - http://www.infoq.com/news/2010/01/subbu-custom-media-types - old article, but still true.

The problem is that `application/json` doesn't really tell you anything about the semantics of the returned JSON.

Can't the server say that it `@Consumes`/`@Produces` both `application/json` and the custom `vnd` type?